kroxylicious-filters/kroxylicious-simple-transform/pom.xml

```diff
@@ -69,5 +69,16 @@
             <artifactId>mockito-junit-jupiter</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-databind</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>io.kroxylicious</groupId>
+            <artifactId>kroxylicious-runtime</artifactId>
+            <scope>test</scope>
+        </dependency>
```

Comment:
The fact I needed to add a dependency on `kroxylicious-runtime`, in order to get an object mapper that understood plug-in loading, so that I could write a test that we could round trip the config too feels a bit wrong. 

What are your thoughts @robobario @k-wall